src/ImageProcessor/Imaging/ResizeLayer.cs

```diff
@@ -62,107 +46,152 @@ public class ResizeLayer
+            this.RestrictedSizes = restrictedSizes;
             this.AnchorPoint = anchorPoint;
         }
 
         /// <summary>
         /// Gets or sets the size.
         /// </summary>
+        /// <value>
+        /// The size.
+        /// </value>
         public Size Size { get; set; }
 
         /// <summary>
-        /// Gets or sets the max size.
+        /// Gets or sets the maximum size.
         /// </summary>
+        /// <value>
+        /// The maximum size.
+        /// </value>
         public Size? MaxSize { get; set; }
 
         /// <summary>
-        /// Gets or sets the restricted range of sizes. to restrict resizing methods to.
+        /// Gets or sets the restricted range of sizes to restrict resizing methods to.
         /// </summary>
+        /// <value>
+        /// The restricted sizes.
+        /// </value>
         public List<Size> RestrictedSizes { get; set; }
 
         /// <summary>
         /// Gets or sets the resize mode.
         /// </summary>
+        /// <value>
+        /// The resize mode.
+        /// </value>
         public ResizeMode ResizeMode { get; set; }
 
         /// <summary>
         /// Gets or sets the anchor position.
         /// </summary>
+        /// <value>
+        /// The anchor position.
+        /// </value>
         public AnchorPosition AnchorPosition { get; set; }
 
         /// <summary>
         /// Gets or sets a value indicating whether to allow up-scaling of images.
-        /// For <see cref="T:ResizeMode.BoxPad"/> this is always true.
+        /// For <see cref="T:ResizeMode.BoxPad" /> this is always true.
         /// </summary>
+        /// <value>
+        ///   <c>true</c> if up-scaling is allowed; otherwise, <c>false</c>.
+        /// </value>
         public bool Upscale { get; set; }
 
         /// <summary>
-        /// Gets or sets the center coordinates.
+        /// Gets or sets the center coordinates (Y,X).
+        /// </summary>
+        /// <value>
+        /// The center coordinates (Y,X).
+        /// </value>
+        [Obsolete("Use the Center property instead.")]
+        public float[] CenterCoordinates
+        {
+            get
+            {
+                return this.Center is PointF center ? new float[] { center.Y, center.X } : null;
+            }
+            set
+            {
+                if (value != null && value.Length == 2)
+                {
+                    this.Center = new PointF(value[1], value[0]);
+                }
+                else
+                {
+                    this.Center = null;
+                }
+            }
+        }
+
+        /// <summary>
+        /// Gets the center coordinates as <see cref="Nullable{PointF}" />.
         /// </summary>
-        public float[] CenterCoordinates { get; set; }
+        /// <value>
+        /// The center coordinates as <see cref="Nullable{PointF}" />.
+        /// </value>
+        public PointF? Center { get; set; }
```

Comment:
This ensures backwards compatibility: keep the original `float[] CenterCoordinates` as obsolete property, while making it easier for other implementations to use the coordinates from the `PointF? Center` property 👍